src/software/thunderscope/robot_communication.py

```diff
@@ -205,8 +213,9 @@ def __enter__(self):
         self.send_estop_state_thread.start()
         self.run_thread.start()
 
-    def __exit__(self):
+    def __exit__(self, type, value, traceback):
```

Comment:
The added parameters don't seem to be used

Reply:
The `__exit__` method is called with [these](https://docs.python.org/3/reference/datamodel.html#object.__exit__) parameters when we exit the `with` statement, so its more correct to keep it this way (i think i was getting an exception in some scenarios because of this but i cant remember..).  It is also consistent with our use of `__exit__` elsewhere in the codebase